modules/coreos-layering-on-cluster-applying-new-configs.adoc

```diff
@@ -0,0 +1,80 @@
+// Modules included in the following assemblies:
+//
+// * post-installation_configuration/coreos-layering.adoc
+
+:_content-type: PROCEDURE
+[id="coreos-layering-on-cluster-applying-new-configs_{context}"]
+= Applying new configurations to on-cluster builds
+
+Any opted-in machine config pools must be paused before you can update any machine configs for that pool. The following example uses a machine config file `layered-os-image-override.yaml` that was created for an opted-in machine config pool.
+
+.Prerequisites
+
+. You have a running {product-title} cluster.
+. You installed the OpenShift CLI (`oc`).
+. You are logged in to the cluster as a user with administrative privileges.
+. You have at least one SSH key pair configured either using the `install-config.yaml` file or a machine config file.
+. You have access to an image registry and those permissions have been added to the global pull secret.
+. You have opted in any machine config pools.
+
+.Procedure
+
+. Pause the machine config pool:
++
+[source,terminal]
+----
+$ oc patch mcp/infra --patch='{"spec": {"paused": true}}' --type=merge
+----
+
+. Apply any new configurations:
++
+[source,terminal]
+----
+$ oc apply -f - <<EOF
+---
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: layered
+  name: new-file-layered
+spec:
+  config:
+    ignition:
+      config: {}
+      security:
+        tls: {}
+      timeouts: {}
+      version: 3.2.0
+    storage:
+      files:
+        - contents:
+        ¦   source: data:,hello%20layered%0A
```

Comment:
`¦` ? Unicode characters in a yaml file, is this correct?  The same goes for the 2 lines below.